src/Mvc/Mvc.Abstractions/src/ModelBinding/BindingInfo.cs

```diff
@@ -169,6 +170,13 @@ public BindingInfo(BindingInfo other)
             break;
         }
 
+        // Keyed services
+        foreach (var fromKeyedServicesAttribute in attributes.OfType<FromKeyedServicesAttribute>())
```

Comment:
Is it possible for more than one `FromKeyedServicesAttribute` to be defined on a parameter? I didn't think so...